EIPS/draft_hardware_and_bandwidth_recs.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Hardware and Bandwidth Recommendations for Validators and Full Nodes
+description: System recommendations for Validators and Full nodes
+author: Parithosh Jayanthi (@parithosh), Kevaundray Wedderburn (@kevaundray), Josh Rudolf (@jrudolf), Dankrad Feist (@dankrad), Justin Traglia (@jtraglia), Ignacio Hagopian (@jsign), George Kadianakis (@asn-d6)
+discussions-to: <URL>
+status: Draft
+type: Informational
+created: 2025-01-26
+requires: <EIP number(s)> # Only required when you reference an EIP in the `Specification` section. Otherwise, remove this field.
+---
+
+## Abstract
+
+This proposal specifies hardware and bandwidth requirements for full nodes and validators. Validators are split into attesters and proposers. The resource heavy component of proposing a block is creating it and broadcasting the necessary data needed to ensure that other attesters view the block as valid. We use the "block builder" to further emphasize this.
```

Comment:
The title and description of the EIP use the word "recommendation". The title description in the PR and the very first sentence of the abstract use the word "requirement". 

Throughout the document these two are used interchangeably and it's not clear what's a recommendation or what is a requirement. For benchmarking reasons or for informed decision-making, is Ethereum meant to use the required system or the recommended one?

Reply:
Will clarify/clean up terminology in the doc tomorrow -- Essentially you can run your node with less than what is listed in the document and it may work fine, but benchmarks etc would be based off of what is listed in there. So merging this PR does not imply immediate changes are needed to ones setup